ansible-satellite6/roles/partition-disk/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+---
+
+- name: disk-free
+  shell: df -P /home | awk 'END{print $NF}'
+  register: disk_free
+  tags:
+  - skip_ansible_lint # "Commands should not change things if nothing needs doing" but this task changes nothing
+
+- name: debug
+  debug: msg="{{ disk_free.stdout }}"
+
+- block:
+  - name: group-name
+    shell: vgs --noheadings -o vg_name | tr -d '  '
+    register: vg
+
+  - name: debug-vg
+    debug: msg="{{ vg }}"
+
+  - name: unmount-home
+    mount:
+      path: /home
+      state: unmounted
+
+  - name: remove-logical-volume
+    lvol:
+      lv: /dev/"{{vg.stdout}}"/home
```

Comment:
Also, it would be ideal if we go with using ansible variables along with `with_dict` option to populate the values of `lv` and `vg`.

```
qe-sat6-feature-rhel7 | SUCCESS => {
    "ansible_facts": {
        "ansible_lvm": {
            "lvs": {
                "lv_root": {
                    "size_g": "193.66", 
                    "vg": "vg_dhcp201"
                }, 
                "lv_swap": {
                    "size_g": "5.84", 
                    "vg": "vg_dhcp201"
                }
            }, 
            "pvs": {
                "/dev/vda2": {
                    "free_g": "0", 
                    "size_g": "199.51", 
                    "vg": "vg_dhcp201"
                }
            }, 
            "vgs": {
                "vg_dhcp201": {
                    "free_g": "0", 
                    "num_lvs": "2", 
                    "num_pvs": "1", 
                    "size_g": "199.51"
                }
            }
        }
    }, 
    "changed": false


Similar to below stuff,

```
- hosts: test
  vars:
    lv:
      vol1:
        vg: cl
        size: 1g
      vol2:
        vg: cl
        size: 5g
      vol3:
        vg: cl
        size: 1g
  tasks:
  - name: Debug mode to see if ansible itself passes variables
    debug: msg="{{ item.key }} --> {{ item.value.vg }} --> {{ item.value.size }}"
    with_dict: "{{ lv }}"

  - name: Test lvol module to build an example lv layout
    lvol:
      vg: "{{ item.value.vg }}"
      lv: "{{ item.key }}"
      size: "{{ item.value.size }}"
      active: yes
      state: present
    with_dict: "{{ lv }}"
```